app/renderer/components/Inspector/LocatedElements.js

```diff
@@ -44,8 +44,8 @@ class LocatedElements extends Component {
             dataSource={locatedElements}
             renderItem={(elementId) =>
               <List.Item type='text'
-                className={locatorTestElement === elementId ? InspectorStyles.searchResultsSelectedItem : ''}
-                onClick={() => setLocatorTestElement(elementId)}
+                {...locatorTestElement === elementId && { className: InspectorStyles.searchResultsSelectedItem }}
```

Comment:
I am not quite sure this is the result we want, let say if `locatorTestElement !== elementId` then the result of the first expression would be `...false`, which does not make sense. Same applied to the second expression

Reply:
I also prefer to use explicit parentheses to such expression as it is not obvious to which element the spread operator is applied, e.g. `...(expr)` vs `...expr`

Reply:
I'll admit, I'm not too familiar with the spread operator, so I was basing the code off [this SO answer](https://stackoverflow.com/a/54039314). Testing showed it was working fine, but if there's a better way, I'd be more than happy to change it.
I definitely agree on the parentheses though, I somehow missed those.

Reply:
it probably works because the spread operator implementation in js is more forgiving in comparison to other languages.

I would try to rewrite this like `...(locatorTestElement === elementId ? { className: InspectorStyles.searchResultsSelectedItem } : {})`